node/resolvers/fileManager/client.ts

```diff
@@ -1,105 +0,0 @@
-import {
```

Comment:
are you sure no other mutations or queries use this client?

Reply:
Necessarily, if a module is using it, since it's not inside this folder, it would have to import one of the modules from `/fileManager`. The search below shows that the only place this is done is exactly the code I've deleted. 

https://github.com/vtex-apps/store-graphql/search?q=fileManager&unscoped_q=fileManager